painel/tops/l2jbrasil.php

```diff
@@ -9,67 +9,20 @@
 //       Brazillian Developer / WebSite: http://www.icpfree.com.br       \\
 //                 Email & Skype: ivan1507@gmail.com.br                  \\
 //=======================================================================\\
-$player_id = md5("ipc".$_SESSION["UsuarioLogin"].$row->top_id);
-$topL2jbrURL = "https://top.l2jbrasil.com/votesystem/?hours=12&player_id={$player_id}&username={$row->top_id}&type=json";
```

Comment:
The top l2jbrasil should use player_id because IPV6 support.

https://www.l2jbrasil.com/topic/144082-adicionado-par%C3%A2metro-player_id-para-rastreio-do-voto